config/config.go

```diff
@@ -10,13 +10,15 @@ type Config struct {
 	RendererURL         string
 	FilterAPIURL        string
 	DatasetAPIURL       string
-	CodeListAPIURL      string
 	HierarchyAPIURL     string
 	DatasetAPIAuthToken string
 	FilterAPIAuthToken  string
 	SearchAPIAuthToken  string
+	ServiceAuthToken    string
```

Comment:
Usually the controllers don't have their own service tokens because they only act on behalf of the user.  Therefore, the service token can be passed to the clients explicitly as `nil` as well.